src/jailer/src/cgroup.rs

```diff
@@ -478,28 +480,45 @@ pub mod test_util {
             Ok(())
         }
 
-        pub fn new() -> std::result::Result<MockCgroupFs, std::io::Error> {
+        pub fn new(
+            mock_proc_mount: &'a str,
```

Comment:
nitpicking
```suggestion
            mock_proc_mounts: &'a str,
```

Reply:
fixed in a9979eda67064f70c1b9f249692b9c0f485d36a1